content/reference/sql/dolt-sql-procedures.md

```diff
@@ -85,6 +86,44 @@ CALL DOLT_ADD('-a');
 CALL DOLT_COMMIT('-m', 'committing all changes');
 ```
 
+## `DOLT_BACKUP()`
+
+Manage server backup endpoints, sync with a backup, or restore from a
+backup. Similar cloud provider configuration as `dolt remote`.
```

Comment:
It looks like only `sync` is supported in SQL currently: https://github.com/dolthub/dolt/blob/main/go/libraries/doltcore/sqle/dfunctions/dolt_backup.go#L68

Reply:
good catch, i saw them all used in bats tests but missed the error checks